components/mpas-albany-landice/src/Registry.xml

```diff
@@ -1471,6 +1477,27 @@ is the value of that variable from the *previous* time level!
 	       />
         </var_struct>
 
+<!-- ================ -->
+
+        <!-- Variables related to time averaging of fluxes -->
+        <var_struct name="timeAveraging" time_levs="1">
+                <var name="timeAccumulatedCoupled" type="real" dimensions="Time" units="s"
+                     description="accumulated time for use in averaging of coupler fields"
+                />
+                <var name="avgCalvingThickness" type="real" dimensions="nCells Time" units="kg m^{-2} s^{-1}"
```

Comment:
As I mentioned elsewhere, this name is misleading, since it's really a flux and not a thickness.

Reply:
As above.

Reply:
I'll address this with the renaming agreed on above.

Reply:
Corrected with https://github.com/E3SM-Project/E3SM/pull/7210/commits/4ae313ea218232fca2061d939c9b36ef023227b3.